costools/add_cos_s_region.py

```diff
@@ -0,0 +1,107 @@
+#! /usr/bin/env python
+
+# Add the S_REGION keyword to raw files
+#
+import glob
+import argparse
+
+from calcos import cosutil
+from astropy.io import fits
+
+__version__ = '0.1'
+def main(dry_run=False):
+    """Add the S-region keyword to all COS FITS files in the current
+    directory that meet the following criteria:
+
+    Filename ends with one of ['rawtag.fits',
+                               'rawtag_a.fits',
+                               'rawtag_b.fits',
+                               'rawaccum.fits',
+                               'rawaccum_a.fits',
+                               'rawaccum_b.fits',
+                               'rawacq.fits']
+
+    The S_REGION keyword will have the value
+    'CIRCLE ra_aper dec_aper diameter'
```

Comment:
See comment below about coordinate frame.

Reply:
Fixed in latest version